migration/20170224-2-copy-collection-configuration-into-database.py

```diff
@@ -113,11 +113,11 @@ def convert_one_click(_db, library):
         name="OneClick"
     )
     library.collections.append(collection)
-    collection.password = basic_token
+    collection.external_integration.password = basic_token
     collection.external_account_id = library_id
-    collection.url = url
-    collection.set_setting("ebook_loan_length", ebook_loan_length)
-    collection.set_setting("eaudio_loan_length", eaudio_loan_length)
+    collection.external_integration.url = url
+    collection.external_integration.set_setting("ebook_loan_length", ebook_loan_length)
+    collection.external_integration.set_setting("eaudio_loan_length", eaudio_loan_length)
```

Comment:
That's fine. I don't think anyone has a OneClick collection in production so probably no one will call this code anyway.